docs/installation/instructions.rst

```diff
@@ -1,8 +1,28 @@
 Installation Instructions
 =========================
 
-Ansible
--------
+Pulp 3 Ansible Galaxy Installation
+----------------------------------
+
+1. In order to be able to run playbook, ensure python3, ansible( >2.2) and libselinux-python are installed:
```

Comment:
this should end in :: for a sphinx literal code block:
http://www.sphinx-doc.org/en/stable/rest.html#source-code